### PR TITLE
Added a file for myself.

### PR DESCRIPTION
**Pull request by htcai**

### CODE REVIEW

File: people/htcai.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Greetings my name is Haitao Cai. I introduced my self [here](https://github.com/cognoma/cognoma/issues/2#issue-163984722).
```
Comment (dhimmel) [inline people/htcai.md]: @htcai, your introduciton link should be changed to `https://github.com/cognoma/cognoma/issues/2#issuecomment-230884763`

Would you like to add another commit to your `addMyName` branch with this change?

File: people/htcai.md [hunk 2]
```diff
@@ -1 +1 @@
-Greetings my name is Haitao Cai. I introduced my self [here](https://github.com/cognoma/cognoma/issues/2#issue-163984722).
\ No newline at end of file
+Greetings my name is Haitao Cai. I introduced my self [here](https://github.com/cognoma/cognoma/issues/2#issuecomment-230884763).
```
Comment (dhimmel) [inline people/htcai.md:1]: 👍

**Comment by dhimmel** [comment]
Awesome, nice job with the update. Will do a squash commit!

**Comment by htcai** [comment]
Thanks, Daniel! I just found how to get the address of my reply from the issue page.

**Pull request merged**